app/api/schema/tickets.py

```diff
@@ -44,6 +44,8 @@ def validate_quantity(self, data):
                 raise UnprocessableEntity({'pointer': '/data/attributes/quantity'},
                                           "quantity should be greater than min-order")
 
+    # Validate discount code?
```

Comment:
use of this comment ?

Reply:
I forgot to mention about this. I wrote it as a placeholder for any possible discount code validations to be added here.